src/main/sarl/fr/utbm/info/ia51/crowdproject/motion/MotionAlgorithms.sarl

```diff
@@ -43,9 +47,23 @@ class KinematicWanderAlgorithm implements WanderAlgorithm {
  */
 class SteeringWanderAlgorithm implements WanderAlgorithm {
 
-	override run(emitter : UUID, currentPosition : Vector2) : MotionQuery {
-		val query = new MotionQuery(emitter, MotionBehavior::STEERING)
-		// TODO
-		return query
+	override run(emitter : UUID, currentPosition : Vector2, orientation : double, velocity : Vector2) : MotionQuery {
```

Comment:
You don't need that. A simple basis vector rotated along the `rotation` parameter is enough. You just have to define what your basis vector will be.